src/test/java/io/ddavison/conductor/FrameworkTest.java

```diff
@@ -26,6 +26,11 @@ public void testSetText() throws Exception {
     }
 
     @Test
+    public void testSetAndValidateText() throws Exception {
+        setAndValidateText("#setTextField", "set and validate text test");
```

Comment:
per your intellij comment, there is an option that will really help with this issue :)

![screen shot 2015-04-01 at 08 26 13](https://cloud.githubusercontent.com/assets/2972876/6940897/c8971420-d848-11e4-9c22-0a3e5ce94d2a.png)

set your `Continuation Indent` to `0` and you'll be golden